assets/translations/Portuguese(Brazil)

```diff
@@ -199,12 +199,12 @@ Skin Colors=Cores da skin
 Skin=Skin
 Space=Espaço
 Suspend timeout=Suspender tempo limite
-SW Clock=
+SW Clock=Relógio do Sistema
```

Comment:
SW Clock stands for SOFTWARE Clock, the same for "Set system's software clock=". Do you think you can change it to make it sound reasonably?